ipfs-paste

```diff
@@ -64,25 +73,38 @@ fi
 if [ $paste ]; then
   test -n "$clipaste" || die "clipboard paste unkown for $(uname)"
   log "paste to ipfs..."
-  fhash=$($clipaste | ipfs add -q)
+
+  date=$(date +"%Y-%m-%dT%H:%M:%SZ")
+  fpath=$(mktemp "/tmp/paste.$date.XXXXXX.txt") ||
+  die "could not 'ktemp /tmp/paste.$date.XXXXXX.txt'"
```

Comment:
Git is very inconsistent about this.
Sometimes the `die` is indented and sometimes not (see https://github.com/git/git/blob/master/git-rebase--interactive.shfor example).

By the way, I use Emacs, and in the shell-script mode it doesn't indent lines following || or &&. I didn't find a simple way to change that, and I think it is not a bug. So I could change it, but it would mean I would either have to fight Emacs or spend a lot of time finding a workaround.